scripts/config/lib/nginx_config.rb

```diff
@@ -13,11 +14,16 @@ def initialize(json_file)
       if hash["origin"][-1] != "/"
         json["proxies"][loc].merge!("origin" => hash["origin"] + "/")
       end
+
+      uri = URI(hash["origin"])
+      json["proxies"][loc]["path"] = uri.path
+      uri.path = ""
+      json["proxies"][loc]["host"] = uri.to_s
```

Comment:
Consider `base` rather than `host` here, perhaps? I nearly recommended using `uri.host` as-is, because I misread this bit of code.